méli-mélo/@sample/index-fr.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: "@ Example pour la création de composant méli-mélo"
 lang: fr
-altLangPage: index-en.html
```

Comment:
Is this a real page that's missing from the repo?

Reply:
Yes and no, that page was only use for testing and do an initial documentation for the méli-mélo feature.